test/karma.ci.conf.js

```diff
@@ -17,8 +17,8 @@ module.exports = function(config) {
         browsers: ['Chrome_travis_ci', 'Firefox', 'PhantomJS'],
         logLevel: config.LOG_INFO,
         preprocessors: {
-            'components/**/*.js': ['coverage']
+            'src/js/**/!(*spec|*helpers).js': ['coverage']
```

Comment:
I don't see the downside to including them, and it is occasionally useful to be able to see that certain tests and/or test files didn't get run. @benpatterson I defer to you on this. What is the policy we've used in other repos?